src/Console/tests/AttributeTest.php

```diff
@@ -62,9 +53,6 @@ public function testCommandWithSymfonyAttribute(): void
             WithSymfonyAttributeCommand::class
         ]));
 
-        $this->assertSame(
-            'Some description text|attribute-with-sf-command-attr',
-            $core->run(command: 'attribute-with-sf-command-attr')->getOutput()->fetch()
-        );
+        self::assertSame('Some description text|attribute-with-sf-command-attr', $core->run(command: 'attribute-with-sf-command-attr')->getOutput()->fetch());
```

Comment:
Why does it squash multiline assertions into inline ones?

Reply:
It rely on AST printing, I can rollback new line manually if needed.

Any coding style I can just run it? I tried exising cs fixer and it seems change many unrelated change

Reply:
> It rely on AST printing

Ah, okay. Let's leave it as is. I'll set up CS Fixer in the future.